frontend/src/concepts/analyticsTracking/useTrackUser.ts

```diff
@@ -4,12 +4,10 @@ import { useAccessReview } from '~/api';
 import { AccessReviewResourceAttributes } from '~/k8sTypes';
 import { IdentifyEventProperties } from '~/concepts/analyticsTracking/trackingProperties';
 
-export const useTrackUser = (
-  username?: string,
-  ssoUserID?: string,
-): [IdentifyEventProperties, boolean] => {
+export const useTrackUser = (username?: string): [IdentifyEventProperties, boolean] => {
   const { isAdmin } = useUser();
-  const [userID, setUserID] = React.useState<string | undefined>(ssoUserID);
+  const { userID } = useUser();
```

Comment:
You can deconstruct this value from the line above:
```
const { isAdmin, userID } = useUser();
```

That being said, where is `userID` coming from. It seems to me that this value is removed from the backend response.

Reply:
`userID` comes via the /api/status call if the backend is sso-enabled.   `useDetectUser()` -> `getUserFulfilled()`

If the cluster has the web-user-id, we get it in the status response like this:

![Bildschirmfoto 2024-10-18 um 15 15 48 Kopie](https://github.com/user-attachments/assets/94407eb5-827d-46fe-b179-3696708c95f5)

Reply:
@pilhuhn I don't see the userID in the response when testing. Make sure you're running your backend changes.

Your change to `backend/src/routes/api/status/statusUtils.ts` specifically removes the `userID` from the object.